step-templates/iis-change-app-offline-online.json

```diff
@@ -46,14 +46,23 @@
         "Octopus.ControlType": "Select",
         "Octopus.SelectOptions": "Offline|Offline\nOnline|Online"
       }
+    },
+    {
+      "Name": "CheckForFile",
+      "Label": "",
+      "HelpText": "Check that either of the files exist first",
```

Comment:
The help text describes the parameter as checking if _either_ of the files exist but the addition to the script checks for **both** and throws an error if they aren't present. Suggest updating this help text to better reflect the change.

Reply:
@KseinyaR-STCU - could you update the `HelpText` property as it is doesn't reflect the code change (as per previous comment)

Reply:
@harrisonmeister  I changed it to "Check that at least one of the files exists first". It only throws an error if both of the files are missing.

Reply:
@KseinyaR-STCU - that is my concern, that the code checks and throws an error if both the files are missing. The Help text implies it only checks if one or the other exists (and not both), whereas it actually only successfully passes that check if both are present, not one (e.g. not either)